ui/style.css

```diff
@@ -0,0 +1,350 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'Gugi', sans-serif, cursive;
```

Comment:
Why the choice of `Gugi`?

Reply:
Do you have this font loaded in the app?

Reply:
No, i dont, was from my previous project, I'll change it